build.sh

```diff
@@ -209,25 +209,33 @@ isMSBuildOnNETCoreSupported()
     # This needs to be updated alongwith corresponding changes to netci.groovy.
     __isMSBuildOnNETCoreSupported=0
```

Comment:
We could certainly change the condition to default to true and then set it to false in a `*)` of the case (have all the other cases be empty).

I think the right long term thing is to probably just try `init-tools.sh` and if that works assume we can build managed code.